src/routes/post.js

```diff
@@ -110,6 +110,14 @@ router.post('/:id/like', loginRequired, checkIdParams, async (ctx) => {
   })
 })
 
+router.post('/:id/react', loginRequired, checkIdParams, async (ctx) => {
+  ctx.body = await Post.react(ctx.identity.id, ctx.request.body.emoji, ctx.params.id)
+})
+
+router.post('/:id/unreact', loginRequired, checkIdParams, async (ctx) => {
+  ctx.body = await Post.unreact(ctx.identity.id, ctx.request.body.emoji, ctx.params.id)
+})
+
```

Comment:
response of this is a raw query response we can have it like:
`
{
    message: 'success'
}
`

Reply:
Like in the comment unreact